operator/config/crd/bases/loki.grafana.com_lokistacks.yaml

```diff
@@ -224,14 +223,15 @@ spec:
                                       description: Regex allows choosing the attributes
                                         by matching a regular expression.
                                       type: string
+                                  required:
+                                  - action
```

Comment:
This change is correct, we just forgot to remove the `omitempty` from one of the `action` definitions. This is part of the OTLP attribute configuration, so this _maybe_ changes soon anyway.